expo-zustand-styled-components/src/components/IssuesTabView/IssuesTabView.styles.ts

```diff
@@ -6,7 +6,7 @@ type ScreenWidth = {
   screenWidth: number | undefined;
 };
 
-export const MainContainer = styled.View<ScreenWidth>`
+export const MainContainer = styled.ScrollView<ScreenWidth>`
   width: ${({ screenWidth }) => (screenWidth > breakpoints.tablet ? '80%' : '100%')};
   margin-horizontal: auto;
   margin-top: 40px;
```

Comment:
I will suggest here padding-top to margin-top
```suggestion
  padding-top: 32px;
```

Reply:
Why padding?